src/extensions/default/QuickView/main.js

```diff
@@ -474,7 +477,8 @@ define(function (require, exports, module) {
 
         // Determine whether or not this URL/path is likely to be an image.
         var parsed = PathUtils.parseUrl(tokenString);
-        var hasProtocol = parsed.protocol !== "";
+        // If the URL has a protocol, check if it's one of the supported protocols
+        var hasProtocol = parsed.protocol !== "" && validProtocols.indexOf(parsed.protocol.trim().toLowerCase()) !== -1;
```

Comment:
Maybe something like `hasValidProtocol` would be a better name for the variable?

Is the empty check (`parsed.protocol !== ""`) still needed?